src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
+
+- `LDAP_TEST_LOGIN`: A login name that will be used to test the user and group filter and validate the configuration. The test login name must be provided as a user id. Example: `activeeon`
+
+- `LDAP_TEST_PASSWORD`: Password associated with the test user.
+
+- `LDAP_ADMIN_ROLES`: List of existing LDAP groups that will be associated with a ProActive administrator role. Multiple groups can be provided separated by commas. Example: `activeeon-admins`
+
+- `LDAP_USER_ROLES`: List of existing LDAP groups that will be associated with a ProActive standard user role. Multiple groups can be provided separated by commas. Example: `activeeon-operators`
+
+[[_compose_PWS_Users_Configuration_Properties]]
+*_PWS Users Configuration Properties_*
+
+In case you would like to create internal user accounts for the ProActive server, you can enable the users configuration in this section:
+
+- `ENABLE_USERS_CONFIGURATION`: Set to true if you would like to create internal ProActive users via login/password. By default, false.
+
+- `PWS_USERS`: A json formatted array where each object defines a set of LOGIN, PASSWORD, and GROUPS of a ProActive internal user. The list of ProActive predefined GROUPS that can be associated with a user are listed and explained following this link: <<_user_permissions>>
+
+[[_compose_External_Database_Configuration_Properties]]
+*_External Database Configuration Properties_*
+
+These parameters are only used when ProActive is started with a Postgres database. It is recommended to not change these parameters unless you would like to change the passwords.
+
+- `ENABLE_EXTERNAL_DATABASE_CONFIGURATION`: Set to true if you would like to start ProActive with a Postgres database. By default, false.
+
+- `POSTGRES_PASSWORD`: Password of the Postgres database user. By default, postgres.
+
+- `DB_DNS`: DNS of the machine hosting the database. By default, it is hosted in the same network as the ProActive server container, hence, the database container name is referenced.
+
+- `DB_PORT`: Database port. By default, 5432.
+
+- `DB_DIALECT`: Database dialect to be used for hibernate communication. By default, PostgreSQL94Dialect.
+
+- `DB_RM_PASS`: Password of the `rm` user in the RM database. By default, rm.
+
+- `DB_SCHEDULER_PASS`: Password of the `scheduler` user in the SCHEDULER database. By default, scheduler.
+
+- `DB_CATALOG_PASS`: Password of the `catalog` user in the CATALOG database. By default, catalog.
+
+- `DB_PCA_PASS`: Password of the `pca` user in the PCA database. By default, pca.
+
+- `DB_NOTIFICATION_PASS`: Password of the `notification` user in the NOTIFICATION database. By default, notification.
+
+==== Docker Compose Configuration with an Embedded HSQL Database
+Once you set the right configuration options in the `.env` file, you can use the Docker Compose configuration provided in the installation package (`docker-compose-hsqldb.yml`) to run the ProActive server container with an embedded HSQL database.
+
```

Comment:
This sentence is not clear. You need to specify what are the "right" configuration options that enable to run the ProActive server container with an embedded HSQL database.
ENABLE_EXTERNAL_DATABASE_CONFIGURATION = false
etc

Reply:
To use the Docker Compose configuration with a PostgreSQL database, ensure that the correct configuration options are set in the .env file beforehand. Additionally, enable the advanced property ENABLE_EXTERNAL_DATABASE_CONFIGURATION.

Once done, you can then proceed with the Docker Compose configuration provided in the installation package docker-compose-pg.yml to launch the ProActive server container alongside an external PostgreSQL database container.

It will start two containers, one for the server, and one for the database.

Reply:
it is done like above